package.json

```diff
@@ -67,25 +82,90 @@
     "eslint-plugin-snakecasejs": "^2.2.0",
     "eslint-plugin-sort-dojo-dependency": "^0.0.1",
     "eslint-plugin-standard": "^4.0.1",
+    "extract-css-chunks-webpack-plugin": "^4.7.5",
+    "extract-loader": "5.1.0",
+    "extract-text-webpack-plugin": "^3.0.2",
+    "file-loader": "^6.0.0",
+    "glob": "7.1.6",
+    "html-loader": "^1.1.0",
+    "html-webpack-inline-source-plugin": "^0.0.10",
+    "html-webpack-plugin": "^4.3.0",
+    "inspectpack": "4.5.2",
+    "less": "^3.11.2",
+    "less-loader": "^6.1.0",
+    "lodash": "^4.17.15",
+    "lodash-amd": "^4.17.15",
     "markdownlint": "0.20.3",
     "markdown-spellcheck": "1.3.1",
     "markdownlint-cli": "0.23.1",
+    "mini-css-extract-plugin": "^0.9.0",
+    "mqpacker": "^7.0.0",
+    "npm-check": "5.9.2",
+    "npm-run-all": "4.1.5",
+    "postcss": "7.0.32",
+    "postcss-assets": "^5.0.0",
+    "postcss-assets-urls": "^0.2.3",
+    "postcss-assets-webpack-plugin": "^3.0.0",
+    "postcss-browser-reporter": "^0.6.0",
+    "postcss-import": "12.0.1",
+    "postcss-loader": "3.0.0",
+    "postcss-preset-env": "6.7.0",
+    "postcss-reporter": "^6.0.1",
+    "postcss-stats-reporter": "^0.4.1",
+    "postcss-url": "^9.0.0",
     "prettier": "^2.0.5",
     "prettier-plugin-html-template-literals": "1.0.5",
+    "recursive-readdir": "^2.2.2",
+    "ref-loader": "1.0.3",
+    "request": "^2.88.2",
+    "resolve-url": "^0.2.1",
+    "resolve-url-loader": "^3.1.1",
+    "sass": "^1.26.7",
+    "sass-loader": "8.0.2",
+    "script-loader": "0.7.2",
+    "style-loader": "^1.2.1",
     "stylelint": "^13.6.0",
     "stylelint-config-standard": "^20.0.0",
     "stylelint-no-unsupported-browser-features": "^4.0.0",
     "stylelint-order": "^4.1.0",
     "stylelint-order-config-standard": "^0.1.3",
-    "stylelint-webpack-plugin": "^2.0.0"
+    "stylelint-webpack-plugin": "^2.0.0",
+    "terser-webpack-plugin": "^3.0.3",
+    "unused-webpack-plugin": "^2.4.0",
+    "url-loader": "^4.1.0",
+    "web-webpack-plugin": "^4.6.7",
+    "webpack": "^4.43.0",
+    "webpack-bundle-analyzer": "^3.8.0",
+    "webpack-chunk-rename-plugin": "^1.0.3",
+    "webpack-cli": "^3.3.11",
+    "webpack-dashboard": "3.2.0",
+    "webpack-debug-plugin": "1.0.0",
+    "webpack-dev-server": "^3.11.0",
+    "webpack-entry-plus": "^1.0.18",
+    "webpack-hasjs-plugin": "^1.0.3",
+    "webpack-i18n-extractor-plugin": "^2.0.7",
+    "webpack-log": "^3.0.1",
+    "webpack-merge": "^4.2.2",
+    "webpack-merge-and-include-globally": "^2.1.24",
+    "webpack-monitor": "1.0.14",
+    "webpack-multiple-themes-compile": "^2.0.0",
+    "webpack-visualizer-plugin": "^0.1.11",
+    "wrapper-webpack-plugin": "^2.1.0"
```

Comment:
Do we really need all these? Or is now a good time to reduce the list to what's required?

Reply:
`webpack-multiple-themes-compile` is required to support dijit themes. The others are already gone.